pyproject.toml

```diff
@@ -43,20 +43,20 @@ pyyaml = "^6.0.1"
 
 # For production, we use poetry to build the python package
 
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+#[build-system]
+#requires = ["poetry-core>=1.0.0"]
+#build-backend = "poetry.core.masonry.api"
 
 # For local development, we use maturin to build the rust library
 
-# [build-system]
-# requires = ["maturin", "setuptools>=42", "wheel"]
-# build-backend = "maturin"
+[build-system]
+requires = ["maturin", "setuptools>=42", "wheel"]
+build-backend = "maturin"
 
-# [tool.maturin]
-# name = "fennel_data_lib"
-# sdist-directory = "python_package"
-# manifest-path = "../server/fennel_data_lib/Cargo.toml"
+[tool.maturin]
+name = "fennel_data_lib"
+sdist-directory = "python_package"
+manifest-path = "../server/fennel_data_lib/Cargo.toml"
```

Comment:
I suppose you'd need to revert this before merging?

Reply:
Yup.

Reply:
yup reverted